spring-integration-core/src/main/java/org/springframework/integration/util/UUIDConverter.java

```diff
@@ -30,15 +32,17 @@
  *
  * @author Dave Syer
  * @author Gary Russell
+ * @author Christian Tzolov
  */
 public class UUIDConverter implements Converter<Object, UUID> {
 
-	public static final String DEFAULT_CHARSET = "UTF-8";
+	public static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
 
+	private static final Pattern UUID_REGEX = Pattern
+			.compile("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$");
 
 	/**
-	 * Convert the input to a UUID using the convenience method
-	 * {@link #getUUID(Object)}.
+	 * Convert the input to a UUID using the convenience method {@link #getUUID(Object)}.
 	 *
```

Comment:
Consider to remove blank lines from method Javadocs since we are already here in changes for this class.